steppy/base.py

```diff
@@ -522,6 +536,20 @@ def _get_steps(self, all_steps):
         all_steps[self.name] = self
         return all_steps
 
+    def _validate_name(self, name, transformer):
```

Comment:
@kamil-kaczmarek as I said before I would simply move the None logic outside and have _create_name done here.